src/screens/AccountRecover.js

```diff
@@ -53,124 +59,121 @@ class AccountRecoverView extends React.Component {
-            />
-            <AccountCard
-              style={{ marginTop: 20 }}
-              address={selected.address || ''}
-              networkKey={selected.networkKey || ''}
-              title={selected.name}
-              seedType={selected.validBip39Seed ? 'bip39' : 'brain wallet'}
-            />
-            <Button
-              buttonStyles={{ marginBottom: 40 }}
-              title="Next Step"
-              onPress={() => {
-                const validation = validateSeed(selected.seed, selected.validBip39Seed);
-
-                if (!validation.valid) {
-                  if (validation.accountRecoveryAllowed){
-                    return Alert.alert(
-                      'Warning:',
-                      `${validation.reason}`,
-                      [
-                        {
-                          text: 'I understand the risks',
-                          style: 'default',
-                          onPress: () => {
-                            this.props.navigation.navigate('AccountPin', {
-                              isWelcome: this.props.navigation.getParam(
-                                'isWelcome'
-                              ),
-                              isNew: true
-                            });
-                          }
-                        },
-                        {
-                          text: 'Back',
-                          style: 'cancel'
-                        }
-                      ]
-                    );
-                  } else {
-                    return Alert.alert(
-                      'Error:',
-                      `${validation.reason}`,
-                      [
-                        {
-                          text: 'Back',
-                          style: 'cancel'
-                        }
-                      ]
-                    );
-                  }
+            value={this.state.seed}
+          />
+          <AccountCard
+            style={{ marginTop: 20 }}
+            address={selected.address || ''}
+            networkKey={selected.networkKey || ''}
+            title={selected.name}
+            seedType={selected.validBip39Seed ? 'bip39' : 'brain wallet'}
+          />
+          <Button
+            buttonStyles={{ marginBottom: 40 }}
+            title="Next Step"
+            onPress={() => {
+              const validation = validateSeed(
+                selected.seed,
+                selected.validBip39Seed
+              );
+
+              if (!validation.valid) {
+                if (validation.accountRecoveryAllowed) {
+                  return Alert.alert('Warning:', `${validation.reason}`, [
+                    {
+                      text: 'I understand the risks',
+                      style: 'default',
+                      onPress: () => {
+                        this.props.navigation.navigate('AccountPin', {
+                          isWelcome: this.props.navigation.getParam(
```

Comment:
not needed anymore (see previous comment)
```suggestion
                          isWelcome: navigation.getParam(
```